Examples/Scripts/Python/full_chain_odd.py

```diff
@@ -318,6 +318,16 @@
             if args.ttbar
             else TruthSeedRanges()
         ),
+        initialSigmas=[
+            1 * u.mm,
+            1 * u.mm,
+            1 * u.degree,
+            1 * u.degree,
+            0.1 * u.e / u.GeV,
+            1 * u.ns,
+        ],
+        initialSigmaPtRel=0.1,
+        initialVarInflation=[1.0] * 6,
```

Comment:
Why the inflation is now set to 1. ? Shouldn't still be large?

Reply:
I increase the values individually above so we do not need to inflate them afterwards